build/Dockerfile

```diff
@@ -1,10 +1,22 @@
-# build stage for building binaries
-FROM golang:1.13.8 as build
+# We cannot use a golang image as building the kubelet requires rsync and that is not present plus there is no easy way
+# to install it.
+FROM registry.access.redhat.com/ubi8/ubi-minimal as build
 LABEL stage=build
+RUN microdnf -y install rsync make git tar findutils diffutils
+RUN mkdir /build/
+
+# Install go 1.15
```

Comment:
I hear that go 1.15 will soon be available on RHEL8. Hoping this is also the case for `ubi8`, at which point we can get rid of building it from source.